### PR TITLE
Bumped BDN version

### PR DESCRIPTION
**Pull request by naricc**
Update BDN version to fix issue with llvm aot.  I believe the new version has to be uploaded somewhere before we merge this?

Fixes: https://github.com/dotnet/performance/issues/2625

### CODE REVIEW

**Comment by LoopedBard3** [comment]
The Capstone package and BDN version have been uploaded.

**Comment by naricc** [comment]
@adamsitnik Does https://github.com/dotnet/performance/pull/2314 need to be reverted as part of thing change? It seems like dotnet/performance will just consume the arguments and BDN will not see them. I'd assume do the revert separately unless it breaks things?

**Comment by naricc** [comment]
Looks like I answered my own question there; reverted.

File: .gitignore [hunk 1]
```diff
@@ -55,7 +55,6 @@ bld/
 
 # Visual Studio 2017 auto generated files
 Generated\ Files/
-**/*Properties/launchSettings.json
```
Comment (LoopedBard3) [inline .gitignore:58]: Was this removal on purpose?

**Review by LoopedBard3 — APPROVED** [review]
LGTM, assuming clean ci run and comment is answered.

File: .gitignore (hunk 1 above)
Comment (naricc) [reply .gitignore:58]: It was removed because I used `git revert` to auto generate a revert commit for  [eng/pipelines/runtime.yml](https://github.com/dotnet/performance/pull/2314), and it had the change adding it to .gitignore. I think we want to keep it though; I will fix that.

**Review by adamsitnik — APPROVED** [review]
LGTM, thank you for your work on this @naricc ! 

@LoopedBard3 thank you for your help!

**Comment by LoopedBard3** [comment]
The sdk_scenario error is unrelated to this change and I don't think the centOS ones are either, though they may be. Regardless, I have an issue tracking the centos errors here: https://github.com/dotnet/performance/issues/2646.

**Comment by lewing** [comment]
This appears to be breaking the wasm pipeline

**Comment by lewing** [comment]
@radical can you link to one of the failures?

**Comment by radical** [comment]
The failure [log](https://dev.azure.com/dnceng/internal/_build/results?buildId=2020876&view=logs&j=4edff57d-39de-55b3-be71-c5e451bc5767&t=c90beb91-5e2c-5c32-0f6f-7cbadbf68287):

```
Precompiling failed for /home/helixbot/work/BDBD0A7F/w/9C5C0885/e/performance/artifacts/bin/for-running/MicroBenchmarks/50a8be8a-1310-4eec-a7ed-9ea351446f5f/bin/net7.0/browser-wasm/publish/BenchmarkDotNet.dll with exit code 134. 
Failed to load methodspec 0x2b000262 due to Could not load file or assembly 'Mono.Posix.NETStandard, Version=1.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' or one of its dependencies.. 
Run with MONO_LOG_LEVEL=debug for more information. 
Unable to compile method 'bool BenchmarkDotNet.Diagnosers.PerfCollectProfiler:TryInstallPerfCollect (BenchmarkDotNet.Validators.ValidationParameters)' due to: 'Could not load file or assembly 'Mono.Posix.NETStandard, Version=1.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' or one of its dependencies.'. 
Unable to compile method 'void BenchmarkDotNet.Diagnosers.PerfCollectProfiler:StopCollection (BenchmarkDotNet.Diagnosers.DiagnoserActionParameters)' due to: 'Could not load file or assembly 'Mono.Posix.NETStandard, Version=1.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' or one of its dependencies.'. 
Unable to compile method 'bool BenchmarkDotNet.Diagnosers.PerfCollectProfiler/<Validate>d__18:MoveNext ()' due to: 'Could not load file or assembly 'Mono.Posix.NETStandard, Version=1.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' or one of its dependencies.'. 
corrupted size vs. prev_size 

Precompiling failed for /home/helixbot/work/BDBD0A7F/w/9C5C0885/e/performance/artifacts/bin/for-running/MicroBenchmarks/50a8be8a-1310-4eec-a7ed-9ea351446f5f/bin/net7.0/browser-wasm/publish/System.Linq.dll with exit code 139. 
Precompiling failed for /home/helixbot/work/BDBD0A7F/w/9C5C0885/e/performance/artifacts/bin/for-running/MicroBenchmarks/50a8be8a-1310-4eec-a7ed-9ea351446f5f/bin/net7.0/browser-wasm/publish/Microsoft.Diagnostics.Tracing.TraceEvent.dll with exit code 139. 
Precompiling failed for /home/helixbot/work/BDBD0A7F/w/9C5C0885/e/performance/artifacts/bin/for-running/MicroBenchmarks/50a8be8a-1310-4eec-a7ed-9ea351446f5f/bin/net7.0/browser-wasm/publish/MicroBenchmarks.dll with exit code 139. 
Precompiling failed for /home/helixbot/work/BDBD0A7F/w/9C5C0885/e/performance/artifacts/bin/for-running/MicroBenchmarks/50a8be8a-1310-4eec-a7ed-9ea351446f5f/bin/net7.0/browser-wasm/publish/System.Private.Xml.dll with exit code 139. 
Precompiling failed for /home/helixbot/work/BDBD0A7F/w/9C5C0885/e/performance/artifacts/bin/for-running/MicroBenchmarks/50a8be8a-1310-4eec-a7ed-9ea351446f5f/bin/net7.0/browser-wasm/publish/System.Private.CoreLib.dll with exit code 139. 
```

**Comment by lewing** [comment]
cc @SamMonoRT @jeffhandley

**Comment by adamsitnik** [comment]
I've sent a "fix": https://github.com/dotnet/BenchmarkDotNet/pull/2154
